### PR TITLE
UserAccount -> UserState + PDA->Account

### LINKED ISSUE #16: Rename to match variables to contracts

### PR DESCRIPTION
**Pull request by vaibhavchellani**
Fix #16

### CODE REVIEW

File: config/genesis.go [hunk 1]
```diff
@@ -64,22 +64,22 @@ func NewGenUserAccount(id, balance, tokenType, nonce, status uint64, publicKey s
 }
 
 type GenesisAccounts struct {
```
Comment (ChihChengLiang) [inline config/genesis.go:66]: Should be `GenesisStates`?
```suggestion
type GenesisStates struct {
```

File: config/genesis.go [hunk 2]
```diff
@@ -64,22 +64,22 @@ func NewGenUserAccount(id, balance, tokenType, nonce, status uint64, publicKey s
...
-	Accounts []GenUserAccount `json:"gen_accounts"`
+	Accounts []GenUserState `json:"gen_accounts"`
```
Comment (ChihChengLiang) [inline config/genesis.go:67]:
```suggestion
	States []GenUserState `json:"gen_states"`
```

File: config/genesis.go [hunk 3]
```diff
@@ -64,22 +64,22 @@ func NewGenUserAccount(id, balance, tokenType, nonce, status uint64, publicKey s
...
 }
 
-func NewGenesisAccounts(accounts []GenUserAccount) GenesisAccounts {
+func NewGenesisAccounts(accounts []GenUserState) GenesisAccounts {
```
Comment (ChihChengLiang) [inline config/genesis.go:70]:
```suggestion
func NewGenesisStates(states []GenUserState) GenesisStates {
```

**Review by ChihChengLiang — APPROVED** [review]
Looks good! I noticed some missed out rename, but maybe no rush to fix it in this PR as it's size is so big now. We can gradually fix them in the next rounds.

**Comment by vaibhavchellani** [comment]
Yeah Ill fix these, there is just so much refactor to do :))

**Pull request merged**